### DISCUSSION TITLE
Map panning start and stop events

### DISCUSSION THREAD

**Discussion by FrozDark** [Mapsui/Mapsui#1323] [Q&A]
Is there anyway I can detect the start panning and stop? 
Especially the end of fling animation

**Comment by charlenni** [comment]
Look at `MapControl.Viewport.ViewportChanged`

**Comment by FrozDark** [comment]
I saw it but it just a property change event of position change, size, resolution and etc...

I would like to detect when the fling animation stops

**Reply by charlenni** [reply]
Try `Navigator.Navigated` and check for `ChangeType.Discrete` as ChangeType. If fling is still running, then ChangeType is `ChangeType.Continous`.

There isn't an event that is called when fling ends.
